yal/main.py

```diff
@@ -206,6 +206,13 @@ async def state_qa_reset_feedback_timestamp_keywords(self):
             "the message early",
         )
 
+    async def state_qa_start_baseline_survey(self):
+        self.save_metadata("assessment_name", "connectedness_v2")
+        self.save_metadata(
+            "assessment_end_state", "state_connectedness_assessment_v2_end"
+        )
+        return await self.go_to_state(AssessmentApplication.START_STATE)
+
```

Comment:
I think it would be better to change line [176](https://github.com/praekeltfoundation/vaccine-eligibility/blob/main/yal/main.py#L176) to point to [state_invitation](https://github.com/praekeltfoundation/vaccine-eligibility/blob/main/yal/surveys/baseline.py#L17) instead. These start survey states are only in main.py for bad historical reasons.